### PR TITLE
Add GitLab Worker

### PR DESCRIPTION
**Pull request by bkaganyildiz**

### CODE REVIEW

File: src/GitLabWorker/worker.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+def receive_from_queue():
+    channel.basic_consume(callback,
+                          queue=QUEUE_NAME,
+                          no_ack=True)
+    print(' [*] Waiting for messages. To exit press CTRL+C')
+    channel.start_consuming()
+
+
+def get_gitlab_users_starred_repos(user_list):
+    for user in list_of_users:
```
Comment (bkaganyildiz) [inline src/GitLabWorker/worker.py]: user_list

File: src/GitLabWorker/worker.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        responses = get(url=GITLAB_API_ENDPOINT.format(user), params=PARAMS).json()
+        for response in responses:
+            print (response['name'])
+            print (response['web_url'])
+
+if __name__ == '__main__':
+    user_list = receive_from_queue()
```
Comment (bkaganyildiz) [inline src/GitLabWorker/worker.py]: no return value user_list always void

File: src/GitLabWorker/worker.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    get_gitlab_users_starred_repos(user_list=user_list)
```
Comment (bkaganyildiz) [inline src/GitLabWorker/worker.py]: redundant job

**Comment by anlcnydn** [comment]
Merged.

**Pull request closed without merging**